sources/packages/web/src/components/institutions/modals/confirmationOfEnrollment/DenyCOE.vue

```diff
@@ -54,7 +54,6 @@
           primaryLabel="Decline enrolment now"
           @primaryClick="submit"
           @secondaryClick="cancel"
-          :disablePrimaryButton="notAllowed"
```

Comment:
Can you please elaborate on why was this removed?

Reply:
same here